src/components/Jam.jsx

```diff
@@ -72,6 +77,42 @@ export default function Jam() {
     setCollaborativeOperationRunning(coinjoinInProgress || makerRunning)
   }, [serviceInfo])
 
+  const reloadSchedule = useCallback(
+    ({ signal }) => {
+      return Api.getTumblerSchedule({ walletName: wallet.name, token: wallet.token })
+        .then((res) => (res.ok ? res.json() : Api.Helper.throwError(res)))
+        .then((data) => {
+          setSchedule(data.schedule)
+        })
+        .catch((err) => {
+          // Not finding a schedule is not an error.
+          // It means a single colalborative transaction is running.
+          // Those have no schedule.
```

Comment:
Not finding a schedule with 404 is not an error. Everything else is an error, isn't it?
Status code can be obtained with `err.response.status` if it is an `JmApiError`.

Reply:
Good point! Updated. ✅